### PR TITLE
Recent updates from Software Verde repo

### PR DESCRIPTION
**Pull request by thesquaregroot**
Includes:

- HF-20220515
  - CHIP: Bigger Script Integer
  - CHIP: Native Introspection Opcodes
- Removal of coinbase rule from HF-20201115
- Clarification of ambiguity of term "var int"
- Fix for transaction signing sequence number length
- Formatting and link fixes

### CODE REVIEW

**Comment by thesquaregroot** [comment]
After looking at the other pull requests, I believe the contains the changes from #39 and #40.

**Comment by melroy89** [comment]
Plus Lots of love!

File: protocol/forks/chips/2022-05-bigger-script-integers.md [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+
+### 128-bit Integers
+
+An [earlier proposal for 128-bit integers](https://gist.github.com/rkalis/eabdbba283f3807c0e38bd677672b6ae) would also enable up to 128-bit arithmetic operations. The larger 128-bit range [may impact worst-case validation performance](https://gist.github.com/bitjson/78967c2affddaf3b7d1ab4dee71a940f#prototype--benchmark-dos-viability), and implementation is likely to be more complex in many computing environments.
+
+While future proposals could further expand the range of VM arithmetic operations, [64-bit math is likely sufficient even for operation on 1/1000th "fractional satoshis"](#limiting-arithmetic-operations-to-8-byte-script-numbers), so further expansion requires additional research.
+
+### NextChain BigNum
+
+[NextChain BigNum](http://nextchain.cash/bignum) would enable up to 4096-bit integer arithmetic, add the `OP_SETBMD`, `OP_BIN2BIGNUM`, and `OP_BIGNUM2BIN` operations, and introduce "type" information to all stack items (with a new `BigNum` type).
```
Comment (sickpig) [inline protocol/forks/chips/2022-05-bigger-script-integers.md:271]: this is a dead url, you need to use www.nextchain.cash/bugnum

File: protocol/forks/chips/2022-05-native-introspection-opcodes.md [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+- [Bitcoin Cash Node (BCHN) Native Introspection Tests](https://gitlab.com/bitcoin-cash-node/bitcoin-cash-node/-/blob/6fff8c761fda0ad15c9752d02db31aa65d58170f/src/test/native_introspection_tests.cpp)
+
+## Evaluation of Alternatives
+
+No alternative native introspection proposals are currently public, but alternative designs for several components of this proposal have been documented in the [Rationale](#rationale) section.
+
+Several past proposals have informed the design of this proposal:
+
+- The [initial `OP_PUSHSTATE`](https://github.com/bitjson/op-pushstate) proposal used a single opcode and templated approach. (The proposal [was withdrawn](https://bitcoincashresearch.org/t/chip-2021-02-add-native-introspection-opcodes/307/15) in favor of this CHIP.)
+- An [implementation of a closely-related `OP_PUSH_TX_STATE`](http://nextchain.cash/op_push_tx_state) exists for the Bitcoin Unlimited node software in the NextChain project.
```
Comment (sickpig) [inline protocol/forks/chips/2022-05-native-introspection-opcodes.md:271]: dead URL, use http://nextchain.cash/op_push_tx_state

File: protocol/forks/chips/2022-05-native-introspection-opcodes.md [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+
+- [Bitcoin Cash Node (BCHN) Merge Request 1208](https://gitlab.com/bitcoin-cash-node/bitcoin-cash-node/-/merge_requests/1208)
+
+### Test Cases
+
+- [Bitcoin Cash Node (BCHN) Native Introspection Tests](https://gitlab.com/bitcoin-cash-node/bitcoin-cash-node/-/blob/6fff8c761fda0ad15c9752d02db31aa65d58170f/src/test/native_introspection_tests.cpp)
+
+## Evaluation of Alternatives
+
+No alternative native introspection proposals are currently public, but alternative designs for several components of this proposal have been documented in the [Rationale](#rationale) section.
```
Comment (sickpig) [inline protocol/forks/chips/2022-05-native-introspection-opcodes.md:266]: Actually there is at least one: 

https://spec.nexa.org/protocol/blockchain/script#introspection
https://spec.nexa.org/doc/push_tx_state

**Comment by sickpig** [comment]
> After looking at the other pull requests, I believe the contains the changes from #39 and #40.

true, even thou we could merge that sooner since they are ready to go, whereas this one need to get at least a bunch of dead urls to fix. you could always rebase on top of `master` w/o much issue.

It would be great to have other devs to chime in.

**Review by sickpig — APPROVED** [review]

**Pull request merged**